static/src/js/components/Map/LayerPicker/__tests__/LayerPicker.test.tsx

```diff
@@ -0,0 +1,158 @@
+import { screen, act } from '@testing-library/react'
+
+import setupTest from '../../../../../../../jestConfigs/setupTest'
+import LayerPicker from '../LayerPicker'
+
+const mockCollectionId = 'C123451234-EDSC'
+
+// Create shared mock functions
+const mockToggleLayerVisibility = jest.fn()
+const mockSetMapLayersOrder = jest.fn()
+const mocksetLayerOpacity = jest.fn()
+
+const mockImageryLayers = {
+  layerData: [
+    {
+      product: 'IMERG_Precipitation_Rate',
+      title: 'Precipitation Rate',
+      colormap: {},
+      isVisible: false,
+      opacity: 1
+    },
+    {
+      product: 'IMERG_Precipitation_Rate_30min',
+      title: 'Precipitation Rate (30-min)',
+      colormap: {},
+      isVisible: true,
+      opacity: 0.5
+    }
+  ],
+  toggleLayerVisibility: mockToggleLayerVisibility,
+  setLayerOpacity: mocksetLayerOpacity,
+  setMapLayersOrder: mockSetMapLayersOrder
+}
+
+const setup = setupTest({
+  Component: LayerPicker,
+  defaultProps: {
+    collectionId: mockCollectionId,
+    imageryLayers: mockImageryLayers
+  },
+  defaultZustandState: {
+    map: {
+      toggleLayerVisibility: mockToggleLayerVisibility,
+      setMapLayersOrder: mockSetMapLayersOrder,
+      setLayerOpacity: mocksetLayerOpacity,
+      mapLayers: {
+        [mockCollectionId]: [
+          {
+            product: 'IMERG_Precipitation_Rate',
+            title: 'Precipitation Rate',
+            isVisible: false,
+            opacity: 1
+          },
+          {
+            product: 'IMERG_Precipitation_Rate_30min',
+            title: 'Precipitation Rate (30-min)',
+            isVisible: true,
+            opacity: 0.5
+          }
+        ]
+      }
+    }
+  }
```

Comment:
LayerPicker does not talk to Zustand, these values are not needed in the test. See my [previous comment](https://github.com/nasa/earthdata-search/pull/1953#discussion_r2375873549)